src/jax.jl

```diff
@@ -23,25 +25,25 @@ end
 
 function (wrap::JaxFunctionWrapper)(args...)
     # TODO: handle multiple outputs
-    out = numpy.array(wrap.jaxfn(mapover(x->jax.numpy.asarray(PyReverseDims(x)), x-> x isa Array, args)...))
-    return reversedims((out))
+    out = (wrap.jaxfn(mapover(x->jax.numpy.asarray(PyReverseDims(x)), x-> x isa Array, args)...))
+    return ReverseDimsArray(DLArray(out, pyto_dlpack))
```

Comment:
Just wondering (haven't really carefully looked into this), why is `ReverseDimsArray` needed here? Would this be equivalent to removing the both the `PyReverseDims` and `ReverseDimsArray`?

Reply:
Right now since we don't have JuliaArray -> DLPack conversion, I have to transfer Julia -> Numpy -> Jax and PyCall by default changes the row/col order. Let me look into exactly what is required to remove this.

Reply:
IIRC from reading the https://github.com/FluxML/ONNX.jl conversion routines, no conversion is required when wrapping the raw pointer if the dimensions are perfectly reversed between Py <-> Julia?

Reply:
Overall we'll need an alternative for https://github.com/JuliaPy/PyCall.jl/blob/master/src/numpy.jl#L175-L187 to fix https://github.com/pabloferz/DLPack.jl/issues/10.

Reply:
We definitely need a `reshape(x, reverse(size(x)))` as well for resolving rowmajor to colmajor issues. But DLArray is doing that now.